schema/home/fetch.js

```diff
@@ -29,6 +29,20 @@ export const featuredGene = (accessToken) => {
   });
 };
 
+export const relatedArtist = (accessToken) => {
+  return gravity.with(accessToken)('me/follow/artists', { size: 10 }).then((follows) => {
+    if (follows.length) {
+      const artist = first(follows).artist;
+      const suggestion_options = { exclude_followed_artists: true, exclude_artists_without_artworks: true, artist_id: artist._id };
+      return gravity.with(accessToken)('me/suggested/artists', suggestion_options).then((suggestions) => {
+        if (suggestions.length) {
+          return first(suggestions);
```

Comment:
Actually that doesn't quite work since we call this method multiple times to get a few pieces of information: title, href, and results.

If you use `sample` here, you might get a different artist for each of those pieces of information.

I think we need to memo-ize that in some smart way.